consensus/src/counters.rs

```diff
@@ -671,3 +675,24 @@ pub static PROPOSER_ELECTION_DURATION: Lazy<Histogram> = Lazy::new(|| {
     )
     .unwrap()
 });
+
+/// Count of the buffer manager retry requests since last restart.
+pub static QUORUM_BATCH_READY_COUNT: Lazy<IntCounter> = Lazy::new(|| {
+    register_int_counter!(
+        "aptos_consensus_quorum_store_batch_count",
```

Comment:
`aptos_consensus_quorum_batch_ready_count`? Also the description below and in the comment above